src/lib/components/ui/ScrollView/__tests__/ScrollView.test.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
```

Comment:
I would like to see functionality test that will check if `scrollBy` is called.

Reply:
Well, #meetoo. I spent over an hour exploring this and as far as I've learned I don't see a way how to test an internal, non-exported module function.

If we export `arrowHandler()` which `scrollBy()` is part of, it could look like this:

```js
describe('functionality', () => {
  it('calls arrowHandler() on arrow buttons click', () => {
    const spy = jest.spyOn(ScrollView, 'arrowHandler');
    const component = mount((
      <ScrollView arrows>
        <span>content</span>
      </ScrollView>
    ));

    component.find('button').at(0).simulate('click');
    component.find('button').at(1).simulate('click');
    expect(spy).toHaveBeenCalledTimes(2);
  });
});
```

But I would like to avoid exports or even conditional exports (using `NODE_ENV` and alike) just for the sake of tests.

I'll be grateful if you show the way out of this tunnel if there is one 🙂.

Reply:
Could we test it by taking a look if `viewportEl.scrollBy` function was called?

Reply:
@adamkudrna I do not see that test. Do we want to leave it without it?